hopr/hopr-lib/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hopr-lib"
-version = "3.0.0"
+version = "3.1.0"
```

Comment:
This should mirror our releases, so should be increase automatically by the CI, not manually.